ACPs/30-avalanche-warp-x-evm.md

```diff
@@ -0,0 +1,216 @@
+```text
+ACP: 30
+Title: Integrate Avalanche Warp Messaging into the EVM
+Author(s): Aaron Buchwald <aaron.buchwald56@gmail.com>
+Status: Proposed
+Track: Standards
+```
+
+## Abstract
+
+Integrate Avalanche Warp Messaging into the C-Chain and Subnet-EVM in order to bring Cross-Subnet Communication to the EVM on Avalanche.
+
+## Motivation
+
+Avalanche Subnets enable the creation of independent blockchains within the Avalanche Network. Each Avalanche Subnet registers its validator set on the Avalanche P-Chain, which serves as an effective "membership chain" for every Subnet on the Avalanche Network.
+
+By providing read access to the validator set of every Subnet on the Avalanche Network, any Subnet can verify an Avalanche Warp Message from any other Subnet including the Avalanche Primary Network within the Avalanche Ecosystem. This completely replaces the need for any point-to-point communication between two Subnets resulting in a light weight protocol that allows seamless, on-demand communication between Subnets.
```

Comment:
```
This completely replaces the need for any point-to-point communication between two Subnets
```

Recommend further clarifying this to be that "validator set info" does not need to be exchanged point-to-point.